### ISSUE TITLE
Use get_all_craft_recipes

### ISSUE THREAD

**Issue by numberZero** [cornernote/minetest-craft_guide#20]
Use `minetest.get_all_craft_recipes` to process recipes already registered to (mostly) remove the need of depending on this mod.

**Comment by numberZero** [comment]
Note that you may also use the `minetest.after(0` trick, as `unified_inventory` does, to process recipes when they are all registered, as well as all aliases.

**Comment by AntumDeluge** [comment]
Does `minetest.after` wait to process until the server is fully loaded?

**Comment by numberZero** [comment]
Yes. The functions queued with `minetest.after` are executed somewhere in the server main loop, but it is started only when all mods are initialized.

**Comment by AntumDeluge** [comment]
Nice! I'll have to use that. Thank you.
